src/models.rs

```diff
@@ -297,6 +308,7 @@ impl AsIdentifier<BlockId> for Block {
             | ToDo { common, .. }
             | Toggle { common, .. }
             | ChildPage { common, .. } => &common.id,
+            | Code { common, .. } => &common.id,
```

Comment:
@nxsy It seems as though you're missing a few match statements, could you run `cargo build` locally and fix them?

Reply:
actually it seems like it is a formatting issue https://github.com/jakeswenson/notion/pull/35/checks#step:4:9
you should be able to address this with `cargo fmt`